core/upgrade/upgrades.php

```diff
@@ -854,6 +854,7 @@ public static function _v_3_16_0_container_updates( $updater ) {
 
 			$data = self::maybe_convert_to_inner_containers( $data );
 			$data = self::maybe_convert_to_grid_container( $data );
+			$data = self::maybe_convert_flex_gap_control( $data );
```

Comment:
@davseve Nice work!!

@TzviRabinovitch @davseve I just noticed that we are iterating through the same data set 3 times with these functions.

```
$data = self::maybe_convert_to_inner_containers( $data );
$data = self::maybe_convert_to_grid_container( $data );
$data = self::maybe_convert_flex_gap_control( $data );
```

Not really the best for performance :-) But we can talk about it, I will approve this task for now.

Reply:
I agree.
We should improve the functionality here.